src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './Footer.scss';
+
+const Footer = () => {
+  return (
+    <>
```

Comment:
I think we don't need React.Fragment (<></>) for wrapper. We have 1 children JSX element.